Drop-In/src/main/java/com/braintreepayments/api/DropInViewModel.java

```diff
@@ -46,7 +46,7 @@ void setSupportedPaymentMethods(List<DropInPaymentMethodType> value) {
     }
 
     void setSupportedCardTypes(List<CardType> value) {
-        supportedCardTypes.postValue(value);
```

Comment:
Why the change here?

Reply:
Since we're already on the main thread we can use `setValue` to decrease the probability of jank. Calling `postValue` is useful when you're on a background thread and you want to update the UI without the app crashing.